### ISSUE TITLE
Feature Request: Add Rhythmbox Layout

### ISSUE THREAD

**Issue by XIncognito10** [quodlibet/quodlibet#2741]
Would it be possible to add the rhythmbox layout to the current layout list? The main things to incorporate would be viewing the list and queue in the left side pane while having the paned browser on the right

**Comment by urielz** [comment]
I think this might be possible whenever https://github.com/quodlibet/quodlibet/issues/1991 gets implemented.

**Comment by declension** [comment]
Before that happens (if ever), we could actually now make a queue sidebar plugin (and hide the main queue I guess) which would get a lot closer..

**Comment by urielz** [comment]
right! Forgot about https://github.com/quodlibet/quodlibet/pull/2538. Thanks for making that happen btw
